build/konflux-patch.sh

```diff
@@ -3,10 +3,10 @@
 set -e
 
 # Gatekeeper Operator image
-stage_operator_img="quay.io/redhat-user-workloads/gatekeeper-tenant/gatekeeper-operator-3-18/gatekeeper-operator-3-18@sha256:6cce8074aff23dde18eb1cee85f6caa9745272f12a39d022481bfb0ea427c19c"
+stage_operator_img="quay.io/redhat-user-workloads/gatekeeper-tenant/gatekeeper-operator-3-18/gatekeeper-operator-3-18@sha256:66ffd39c12a2260151bf98b28ff1870920c40a8fc7e690da99ddbfd24fdca4c4"
 operator_img="registry.redhat.io/gatekeeper/gatekeeper-rhel9-operator@${stage_operator_img##*@}"
 # Gatekeeper image
-stage_gatekeeper_img="quay.io/redhat-user-workloads/gatekeeper-tenant/gatekeeper-operator-3-18/gatekeeper-3-18@sha256:ba92c0f6cb661626e076588a9cdf1427f99fc0fed18f14506a680cc4136c7083"
+stage_gatekeeper_img="quay.io/redhat-user-workloads/gatekeeper-tenant/gatekeeper-operator-3-18/gatekeeper-3-18@sha256:gatekeeper-3-18@sha256:6c0563da0384890aaa6f9085f30f16b7a5f58363b6660f83348cfcf9d2555273"
```

Comment:
doubled `gatekeeper-3-18@sha256`
```suggestion
stage_gatekeeper_img="quay.io/redhat-user-workloads/gatekeeper-tenant/gatekeeper-operator-3-18/gatekeeper-3-18@sha256:6c0563da0384890aaa6f9085f30f16b7a5f58363b6660f83348cfcf9d2555273"
```